paasta_tools/cli/cmds/validate.py

```diff
@@ -317,17 +331,40 @@ def path_to_soa_dir_service(service_path):
     return soa_dir, service
 
 
-def validate_tron(service_path):
+def validate_tron(service_path: str, verbose: bool = False) -> bool:
     soa_dir, service = path_to_soa_dir_service(service_path)
     returncode = True
 
     for cluster in list_tron_clusters(service, soa_dir):
         if not validate_tron_namespace(service, cluster, soa_dir):
             returncode = False
+        elif verbose:
+            # service config has been validated and cron schedules are safe to parse
+            service_config = load_tron_service_config(service, cluster)
+            for config in service_config:
+                cron_expression = config.get_cron_expression()
+                if cron_expression:
+                    print_upcoming_runs(config, cron_expression)
 
     return returncode
 
 
+def print_upcoming_runs(config: TronJobConfig, cron_expression: str) -> None:
+    print(info_message(f"Upcoming runs for {config.get_name()}:"))
+
+    config_tz = config.get_time_zone() or "US/Pacific"
```

Comment:
suggestion: we could extract `"US/Pacific"` to a `DEFAULT_TZ` constant in tron_tools.py